rmarkdown.Rmd

```diff
@@ -157,7 +248,31 @@ model <- lm(Sepal.Length ~ Sepal.Width, data = iris)
 coef(model)
 ```
 
-You can use inline R code too, to insert numbers directly into the document.
+
+#### Exercise 3: adding plots and tables to your results section
+
+Time: 10 min
+
+You now need to create a chunck of code in the **Results section**. If you have any plot from your own research that you want to create feel free to do it so. Otherwise you can just use the iris data:
+
+1. Plot Sepal.length versus Sepal.Width. Include a caption and choose 6 X 10 dimensions.  
+2. Print a table with the mean and sd of each column of the iris dataset. Remember to output just the table and not the code! Use the following code:
+
+```
+#install.packages("dplyr")
+library(dplyr) 
+iris %>%
+  group_by_("Species") %>%
+  summarise_each_(funs=funs("mean", "sd"), vars=c("Sepal.Length", "Sepal.Width"))
```

Comment:
Nice use of the `_`! Though, the attendees will likely not have encountered this style of using dplyr. Plus, the `_` ending has been deprecated in favour of using the rlang package and using tidy evaluation. So now, it would look like:
```
#install.packages("dplyr")
library(dplyr) 
iris %>%
  group_by("Species") %>%
  summarise_at(.vars = c("Sepal.Length", "Sepal.Width"), .funs = funs("mean", "sd"))
```
So you no longer need to use the `_`! It all does things under the hood.